it/uk/gov/hmrc/incometaxsubscriptioneligibility/connectors/GetControlListConnectorISpec.scala

```diff
@@ -15,6 +18,11 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 class GetControlListConnectorISpec extends ComponentSpecBase {
 
+  override lazy val app: Application = new GuiceApplicationBuilder()
+    .in(Environment.simple(mode = Mode.Dev))
+    .configure(config)
```

Comment:
this config is only used here, might be worth moving it from ComponentSpecBase also

Reply:
my bad, overrode the config in AuditServiceISpec instead rather than adding this line in per test.